Book.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * GoogleRichCards
+ * Google Rich Cards metadata generator for Books
+ *
+ * PHP version 5.4
+ *
+ * @category Extension
+ * @package  GoogleRichCards
+ * @author   Igor Shishkin <me@teran.ru>
```

Comment:
File author should be here :-)

Reply:
It is already what is there, no?

Reply:
No, you created the file so you're the author :)